pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md

```diff
@@ -147,7 +152,7 @@ A contrario, si vous souhaitez installer manuellement votre site, munissez-vous
 >
```

Comment:
@Kazarus4, edition of line 113 is missing on FR subs

Reply:
I don't understand this point

Reply:
Sorry, line 109

Reply:
done